server/integration/infinispan/src/main/resources/subsystem-templates/infinispan-core.xml

```diff
@@ -141,7 +141,7 @@
          <cache-container name="local" default-cache="default">
             <global-state />
             <local-cache name="default">
-               <remote-store cache="default" socket-timeout="60000" tcp-no-delay="true" shared="true" hotrod-wrapping="true" purge="false" passivation="false">
+               <remote-store cache="default" socket-timeout="60000" tcp-no-delay="true" shared="true" hotrod-wrapping="true" protocol-version="2.0" purge="false" passivation="false">
```

Comment:
Better omit the protocol version or put the lastest `2.6`. If using `2.0`, the rolling  ugrade process will not use the remote iterator and will fall back to the (bkoen) legacy behaviour